modules/tinymce/src/core/test/ts/browser/caret/LineUtilsTest.ts

```diff
@@ -1,35 +1,47 @@
-import { LegacyUnit } from '@ephox/mcagar';
-import { Pipeline } from '@ephox/agar';
-import Env from 'tinymce/core/api/Env';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Body, Element, Insert, InsertAll, Remove, SelectorFind } from '@ephox/sugar';
 import * as LineUtils from 'tinymce/core/caret/LineUtils';
-import { UnitTest } from '@ephox/bedrock-client';
+import { getClientRects } from 'tinymce/core/dom/Dimensions';
 
-UnitTest.asynctest('browser.tinymce.core.LineUtilsTest', function (success, failure) {
-  const suite = LegacyUnit.createSuite();
+const rect = function (x, y, w, h) {
```

Comment:
Convert to arrow function?

Reply:
It's old code I didn't touch, but might as well... I'll add types as well.